src/UI/NewsSourceModal.jsx

```diff
@@ -11,14 +11,18 @@ import {
 } from "@mui/material";
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
+import { useLocalStorage } from "../service/useLocalStorage";
 
 const NewsSourceModal = ({ isOpen, toggleModal }) => {
   const navigate = useNavigate();
-  const [sources, setSources] = useState({
+
+  const defaultSources = {
```

Comment:
Cool!

One improvement here, move the static constants like this one out of the component body, then it is not recreated on each re-render cycle

Another good practice - use UPPERCASE for static constants (constants that never change), so `DEFAULT_SOURCES`